chapters/pt/chapter1/10.mdx

```diff
@@ -0,0 +1,68 @@
+# Questionário de fim de capítulo
+
+Este capítulo cobriu muito terreno! Não se preocupe se você não entendeu todos os detalhes; os próximos capítulos o ajudarão a entender como as coisas funcionam debaixo do capô.
+
+Primeiro, porém, vamos testar o que você aprendeu neste capítulo!
+
+### [](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/10.mdx#1-explore-the-hub-and-look-for-the-roberta-large-mnli-checkpoint-what-task-does-it-perform)1. Explore o Hub e olhe para o checkpoint `roberta-large-mnli` . Que tarefa ele executa?
+
+<Question choices={[ { text: "Summarização", explicação: "Olhe novamente na página <a href="[https://huggingface.co/roberta-large-mnli">roberta-large-mnli](https://huggingface.co/roberta-large-mnli%22%3Eroberta-large-mnli)." }, { text: "Classificação de texto", explicação: "Mais precisamente, ele classifica se duas ou mais sentenças estão logicamente conectadas entre três rótulos (contradição, neutro, vinculação) — uma tarefa também chamada de _inferência de linguagem natural_.", correct: true }, { text: "Geração de texto", explicação: "Olhe novamente na página <a href="[https://huggingface.co/roberta-large-mnli">roberta-large-mnli](https://huggingface.co/roberta-large-mnli%22%3Eroberta-large-mnli)" } ]} />
```

Comment:
Copy the data from the raw file, this will keep the tabs, as well as ensure that the files will not break due to the links.

This will probably fix what is currently not working in CI
```
<Question
	choices={[
		{
			text: "Summarização",
			explain: "Olhe novamente na página <a href=\"https://huggingface.co/roberta-large-mnli\">roberta-large-mnli</a>."
		},
		{
			text: "Classificação de texto",
			explain: "Mais precisamente, ele classifica se duas ou mais sentenças estão logicamente conectadas entre três rótulos (contradição, neutro, vinculação) — uma tarefa também chamada de <em>inferência de linguagem natural</em>.",
			correct: true
		},
		{
			text: "Geração de texto",
			explain: "Olhe novamente na página <a href=\"https://huggingface.co/roberta-large-mnli\">roberta-large-mnli</a>."
		}
	]}
/>
```

Reply:
Thank you for every review, @johnnv1 . asap i'll fix these issues.